eZ/Publish/Core/FieldType/FieldType.php

```diff
@@ -198,7 +198,7 @@ public function applyDefaultValidatorConfiguration( &$validatorConfiguration )
             foreach ( $configurationSchema as $settingName => $settingConfiguration )
             {
                 // Check that a default entry exists in the configuration schema for the validator but that no value has been provided
-                if ( !isset( $validatorConfiguration[$validatorName][$settingName] ) && isset( $settingConfiguration["default"] ) )
+                if ( !isset( $validatorConfiguration[$validatorName][$settingName] ) && array_key_exists( "default", $settingConfiguration ) )
```

Comment:
Explanation for this change: we now have `null` as default value in a few places to be compatible with Symfony Form component, which makes `isset()` return false while we do need the default value.